detox/src/ios/expect.js

```diff
@@ -164,20 +171,17 @@ class Interaction {
 class ActionInteraction extends Interaction {
   constructor(element, action) {
     super();
-    //if (!(element instanceof Element)) throw new Error(`ActionInteraction ctor 1st argument must be a valid Element, got ${typeof element}`);
-    //if (!(action instanceof Action)) throw new Error(`ActionInteraction ctor 2nd argument must be a valid Action, got ${typeof action}`);
-    this._call = invoke.call(element._call, 'performAction:', action._call);
-    // TODO: move this.execute() here from the caller
+
+    this._call = GreyInteraction.performAction(callThunk(element), callThunk(action));
```

Comment:
@rotemmiz this is currently the only successful invocation of an instance method, I need to find out why the others fail, but I guess it's a call side problem

Reply:
Is this resolved ?